### PR TITLE
Issue 945

### PR DESCRIPTION
**Pull request by zmcNotafraid**
Return transactions if native_transfer receiver is this user address

### CODE REVIEW

**Comment by Keith-CY** [comment]
Please add a brief description of this PR so we won't miss it in commit history

File: lib/godwoken_explorer/graphql/resolovers/transaction.ex [hunk 1]
```diff
@@ -165,11 +165,24 @@ defmodule GodwokenExplorer.Graphql.Resolvers.Transaction do
...
-            t.to_account_id == ^to_account.id or t.from_account_id == ^from_account.id
-          )
+          base_query =
+            query
+            |> where(
+              [t],
+              t.to_account_id == ^to_account.id or t.from_account_id == ^from_account.id
+            )
+
+          if to_account.type == :eth_user do
```
Comment (Keith-CY) [inline lib/godwoken_explorer/graphql/resolovers/transaction.ex:175]: Why not limit the query to `to_account.type == :polyjuice_creator`

Comment (Keith-CY) [inline lib/godwoken_explorer/graphql/resolovers/transaction.ex:175]: Why not limit query to `to_account.type == :polyjuice_creator`

Comment (zmcNotafraid) [reply lib/godwoken_explorer/graphql/resolovers/transaction.ex:175]: For native_transfer query, you need to pass from_account and to_account both as eth_user account

> Why not limit query to `to_account.type == :polyjuice_creator`

Comment (Keith-CY) [reply lib/godwoken_explorer/graphql/resolovers/transaction.ex:175]: Got it

**Review by Keith-CY — APPROVED** [review]

**Review by Naupio — APPROVED** [review]

**Pull request merged**